docs/02_DOCKER.md

```diff
@@ -1,30 +1,51 @@
 # Docker Setup
 
-## Retrieve Meta data.  Feel free to re-run  to update the data.  Set the correct username/password to match your credentials.
+## Retrieve Enrichment Metadata
 
-SCIENCE_USER='user' SCIENCE_PWD='secret' ./initialize_docker_data.sh
+Run the following to download Science Registry and GeoIp data. You may re-run this at anytime to update these databases.
 
-## Build base images
+```sh
+./initialize_docker_data.sh
+```
 
-### Build using Dev Release: 
+## Build Base Images
+
+### Build Using Source Code
+If you would like to build the *importer* container using the version of the pipeline scripts found in this GitHub repo then run the following:
 
 ```sh 
 docker-compose -f docker-compose.build.yml build
 ```
 
-### Build using Production Release: 
-copy the env.template to .env
+### Build Using Production RPM Release 
+You may also build the *importer* container using the RPM published in the GRNOC yum repo as opposed to this source code. **NOTE: The RPM may have older code than is found in this GitHub repository.**
```

Comment:
NOTE: The RPM may have older code than is found in this GitHub repository. -- that reads odd to me, but maybe that's just me.